homeassistant/components/litejet/__init__.py

```diff
@@ -1,49 +1,71 @@
 """Support for the LiteJet lighting system."""
-from pylitejet import LiteJet
+import asyncio
+import logging
+
+import pylitejet
 import voluptuous as vol
 
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
 from homeassistant.const import CONF_PORT
-from homeassistant.helpers import discovery
+from homeassistant.core import HomeAssistant
 import homeassistant.helpers.config_validation as cv
 
-CONF_EXCLUDE_NAMES = "exclude_names"
-CONF_INCLUDE_SWITCHES = "include_switches"
+from .const import DOMAIN, PLATFORMS
 
-DOMAIN = "litejet"
+_LOGGER = logging.getLogger(__name__)
 
 CONFIG_SCHEMA = vol.Schema(
-    {
-        DOMAIN: vol.Schema(
-            {
-                vol.Required(CONF_PORT): cv.string,
-                vol.Optional(CONF_EXCLUDE_NAMES): vol.All(cv.ensure_list, [cv.string]),
-                vol.Optional(CONF_INCLUDE_SWITCHES, default=False): cv.boolean,
-            }
-        )
-    },
+    {DOMAIN: vol.Schema({vol.Required(CONF_PORT): cv.string})},
     extra=vol.ALLOW_EXTRA,
 )
 
 
 def setup(hass, config):
     """Set up the LiteJet component."""
+    if not hass.config_entries.async_entries(DOMAIN) and DOMAIN in config:
```

Comment:
```suggestion
    if DOMAIN in config and not hass.config_entries.async_entries(DOMAIN):
```

slightly more efficient